client/src/components/dashboard.tsx

```diff
@@ -44,6 +44,8 @@ class DashComponent extends React.Component<DashProps> {
         </div>
         <div style={{ float: "right", padding: "1px", margin: "30px" }}>
           <ConnectionQualityComponent quality={100} />
+          <Temperature value = {this.props.sensors.temperature}/>
+
```

Comment:
Minor: remove extra space here. Remember to use Prettier for formatting (short cut shift + alt + f to format)